### ISSUE TITLE
WEM reboots sporadically

### ISSUE THREAD

**Issue by deadrabbit87** [Tinkerforge/esp32-firmware#250]
As described in the issue from [evcc](https://github.com/evcc-io/evcc/issues/8444), my WEM is sporadically rebooting. 

Therefore it is loosing the phase switching request from evcc. 

[debug-report-wem-26ud-2023-06-13T18-41-18-371.txt](https://github.com/Tinkerforge/esp32-firmware/files/11746567/debug-report-wem-26ud-2023-06-13T18-41-18-371.txt)

**Comment by MattiasTF** [comment]
Ich hätte da ein paar Fragen zu deinem Setup:
Sehe ich das Richtig, dass du MQTT eingerichtet hast, um die WEM per evcc zu steuern und gleichzeitig Zählerwerte (nur power) in den WEM zu schieben, damit du eine bunte Grafik bekommst? Zusätzlich setzt du den Zählerstatus (meter/state_update) per HTTP-API? Falls ja, wie häufig tust du das und wie hast du das automatisiert?

**Comment by deadrabbit87** [comment]
Hallo, 

Ja, das ist der Fall. Via Node-Red schreibe ich die Werte vom meinem Kostal Smart Energy Meter (KSEM) an die HTPP-API vom WEM: 
`.../meter/values`

Der Flow ist auch [hier ](https://www.tinkerunity.org/topic/11399-warp-energy-manager/?do=findComment&comment=50776) im Forum. 

Das passiert einmal pro Sekunde. Ich hatte das drin, weil die Steuerung mit EVCC zuerst noch nicht funktioniert hat. Wäre jetzt aber obsolet. 

MQTT ist wegen EVCC eingerichtet.

**Comment by MattiasTF** [comment]
Ich habe leider kein Node-Red und versuche gerade, die JSON-Datei zu verstehen:

- Der obere Flow liest den Stromzähler aus und sendet den Leistungswert einmal pro Sekunde an `meter/values` und nicht `meter/values_update`?

- Der untere Flow fragt `meter/state` ab, und wenn das nicht `{"state":2,"type":4}` ist, wird es per `meter/state_update` auf genau das gesetzt. Wie häufig wird das ausgeführt? Auch einmal pro Sekunde?

**Comment by deadrabbit87** [comment]
> * Der untere Flow fragt `meter/state` ab, und wenn das nicht `{"state":2,"type":4}` ist, wird es per `meter/state_update` auf genau das gesetzt. Wie häufig wird das ausgeführt? Auch einmal pro Sekunde?

Nein, einmal pro Minute. Der obere Flow ist richtig so, ja. Hier wäre auch noch ein Screenshot: 
![grafik](https://github.com/Tinkerforge/esp32-firmware/assets/24924801/9d9e967a-f75d-41a2-b5a4-b9baa9e9c3f0)

**Comment by poohnet** [comment]
Dieser sporadische Reboot tritt seit einiger Zeit auch bei meiner "WARP on Steroids" auf und auch ich sende die Zählerwerte periodisch per Node-RED.

So sieht der zugehörige Callstack des Crashs aus:
![image](https://github.com/Tinkerforge/esp32-firmware/assets/7781206/6ee42d7d-79ee-435b-af87-184eab635b33)

**Comment by deadrabbit87** [comment]
Seit dem deaktivieren von Node Red sind meine Neustarts jetzt weg. WEM läuft bisher seit dem 15.06.2023 15:00 Uhr durch.

**Comment by rtrbt** [comment]
Bitte nochmal mit Firmware 2.1.3 testen. Das Problem sollte jetzt weg sein.
